core/src/main/scala/tapir/Api.scala

```diff
@@ -0,0 +1,24 @@
+package tapir
+import tapir.Api.Info
+
+case class Api(info: Info)
```

Comment:
I'm not sure if this should be in the core module. It's only ever going to be used by docs, no?

Also, why `case class Api(Info)`, and not just `case class ApiInfo(...)` ?